worlds/messenger/docs/setup_en.md

```diff
@@ -18,17 +18,17 @@ Read changes to the base game on the [Game Info Page](/games/The%20Messenger/inf
 3. Click on "The Messenger"
 4. Follow the prompts
 
+These steps can be followed in the future to launch the game and check for mod updates.
```

Comment:
"in the future" seems out of place. Relative to what? If meaning "from now on", proposition: 
"These steps can also be used to launch the game and check for mod updates." 
(A suggestion, not required)

Reply:
I agree with neocerber here. I agree with their suggested sentence, though I'd probably change "used" to "followed"